### PR TITLE
Content Developers Progress and Report

### PR DESCRIPTION
**Pull request by Avinashvytla351**

### CODE REVIEW

File: Server/controllers/question.controller.js [hunk 1]
```diff
@@ -411,7 +412,7 @@ exports.createTutorialsExcel = (req, res) => {
               }
 
               question = new Question({
-                questionId: "IARE" + (currQuestions + (i + 1)).toString(),
+                questionId:(isIdPresent)?data[i].questionId : "IARE" + (currQuestions + (i + 1)).toString(),
```
Comment (nightsailor) [inline Server/controllers/question.controller.js:415]: we don't want this change for now!

File: Server/controllers/participationTut.controller.js [hunk 2]
```diff
@@ -211,3 +226,71 @@ exports.findUserPart = (result, callback) => {
       return callback("Error retrieving data", null);
     });
 };
+exports.findContentDevSolved = (req,res)=>{
+  Participation.find({username:req.params.username})
+  .then((participation)=>{
+    let arr=[]
+    let sliced=req.params.username.substr(7)
+    Participation.findOneAndUpdate({username:req.params.username,courseId:"IARE_PY",practiceSolved:{$exists:false}},
```
Comment (nightsailor) [inline Server/controllers/participationTut.controller.js:234]: https://www.mongodb.com/docs/manual/reference/operator/query/in/

See if below is possible,

```suggestion
    Participation.findOneAndUpdate({username:req.params.username,courseId: { $in: ["IARE_PY", "IARE_C", "IARE_CPP", "IARE_JAVA"]},practiceSolved:{$exists:false}},    
```

File: Server/controllers/question.controller.js (hunk 1 above)
Comment (Avinashvytla351) [reply Server/controllers/question.controller.js:415]: that is for practice questions id purpose without that code won't work
coz for content developer practice questions there is a different format like 535001, 535002 so on. 
for other questions everything is same if the questions are for practice only then have to use this part

**Comment by nightsailor** [comment]
Thanks for the great PR!

**Pull request merged**